src/eterna/ui/VToolbar.ts

```diff
@@ -0,0 +1,81 @@
+// kkk VToolBar.ts --- side tool bar for 3d View
```

Comment:
This file doesn't seem to be used anywhere (maybe it used to). I assume it should be removed.

Reply:
I removed this file